cmd/manager/main.go

```diff
@@ -32,36 +32,36 @@ import (
 )
 
 func main() {
-	// the following line exists to make glog happy, for more information, see: https://github.com/kubernetes/kubernetes/issues/17162
+	// the following line exists to make klog happy, for more information, see: https://github.com/kubernetes/kubernetes/issues/17162
```

Comment:
you need a `klog.InitFlags(nil)` here. otherwise the flags won't get registered. please test the flags too (example set `--log-dir` to see if you can change the directory, set `--log-file` to see if you can send the logs to a file and `--v` to make sure you can adjust the verbosity level)

Reply:
I added it